eds/blocks/partners-navigation/features/profile/dropdown.css

```diff
@@ -89,6 +89,10 @@
   font-size: 14px;
   text-decoration: underline;
   /* PARTNERS_NAVIGATION START */
+  /* MWPW-185175 - Investigate Profile dropdown view account */
```

Comment:
@slobodankajovicic we can merge multiple changes into one * PARTNERS_NAVIGATION START */ block.
anyway: do we really need to add capitalization? we sho9uld stick to default Mil/adobe.com styling unless it's obviously broken.